src/point_one/rtcm/rtcm_framer.h

```diff
@@ -0,0 +1,194 @@
+/**************************************************************************/ /**
+ * @brief RTCM3 message framer.
+ * @file
+ ******************************************************************************/
+
+#pragma once
+
+#include <cstddef> // For size_t
+#include <cstdint>
+
+#include "point_one/fusion_engine/common/portability.h" // For macros.
+
+namespace point_one {
+namespace rtcm {
+
+/**
+ * @brief Frame and validate incoming RTCM3 messages.
```

Comment:
Minor note: technically this is RTCM 10403.3, which is just often shorthanded to RTCM 3.